.circleci/config.yml

```diff
@@ -4,47 +4,35 @@
   jobs:
     unit_tests:
       docker:
-      - image: circleci/golang:1.12.5
+      - image: circleci/golang:1.16
       environment:
         KUBEBUILDER_CONTROLPLANE_START_TIMEOUT: "60s"
       steps:
       - checkout
       - setup_remote_docker
-      - run: curl -L -O https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.0.0-alpha.4/kubebuilder_2.0.0-alpha.4_linux_amd64.tar.gz
-      - run: tar -zxvf kubebuilder_2.0.0-alpha.4_linux_amd64.tar.gz
-      - run: mv kubebuilder_2.0.0-alpha.4_linux_amd64 kubebuilder && sudo mv kubebuilder /usr/local/
-      - run: export PATH=$PATH:/usr/local/kubebuilder/bin
```

Comment:
@avenging why are you removing this?

Reply:
If I remember right this is because "make test" now downloads all the tools needed (etchd, kubeapiserver and kubectl) these get put the in a "testbin" folder. The only tool not in added is kubebuilder, but I don't think that is needed for the CI pipeline.